app/assets/styles/styles.css

```diff
@@ -170,3 +170,26 @@ a {
     font-size: 1.25rem;
   }
 }
+
+.donate {
+  height: 300px;
+  font-size: 1.5rem;
+  margin-left: 2em;
+  margin-right: 2em;
+  background-image: linear-gradient(rgba(21, 21, 78, 0.5), rgba(21, 21, 78, 0.6)), url("/images/newsletter/salvia-team.webp");
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat;
+  border: 5px solid #096a97;
+  color: #fff;
+  text-align: center;
+}
+
+.donate__btn {
+  font-size: 1.5rem;
+  font-weight: bold;
+  text-decoration: none;
```

Comment:
I believe this is applied at the global level. Let's remove it for now. If a line appears, I am confident it will go away once we merge everything together :)